src/components/detailed-card/DetailCard.js

```diff
@@ -1,41 +1,59 @@
-import './detailCard.css'
-export default function DetailCard() {
+import "./detailCard.css";
+import { useCartContext } from "../../contexts/cart-context";
+import { useWishContext } from "../../contexts/wishlist-context";
+
+export default function DetailCard({ prod }) {
```

Comment:
prod can be destructed more here itself it will increase the readabilty.
Example:  
{prod: { image, price, quantity}}

Reply:
Yes i tried to do that but thn i'm not able to pass prod in payload?